content/docs/reference/fields.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Fields
+id: fields
+last_edited: '2022-06-15T15:51:56.737Z'
+next: /docs/reference/templates
+prev: /docs/reference/collections
+---
+
+<!-- # next: /docs/reference/schema -->
+
+Fields define the shape of the content and the user input. There are [many types of fields](/docs/reference/types) each with its own input and type.
+
+
+Although some fields have more properties here is a list of common ones that are used.
+
+## Definition
+
+| Property     | Description              |
+|--------------|--------------------------|
+| `name` | The name of the field |
+| `type` | The [type of the field](/docs/reference/types/) to be used |
+| `label` | A human friendly label that will be displayed to the user (*optional* and will default to `name`)|
+| `required` | If `true`, the collection can not be saved without this field present (*optional*, defaults to `false`) | 
+| `isTitle`  | Denote a field as the title of a collection. [See below for more details](#istitle) (*optional*, defaults to `false`)|
+| `isBody`  | If `true` this field will be used as the body of the document.  [See below for more details](#isbody) (*optional*, defaults to `false`) |
+| `ui`  | Used to extend the user interface of the field and the field behaver. See [extending tina section](/docs/extending-tina/overview/) for more information (*optional*) |
```

Comment:
the overview of extending-tina docs doesn't really address how to use the ui property - should this link to a more meaningful page? or do we need a more meaningful page for extending-tina ui first?